.github/workflows/test.yml

```diff
@@ -45,6 +45,7 @@ on:
 
 env:
   PRISMA_TELEMETRY_INFORMATION: 'prisma test.yml'
+  NODE_OPTIONS: '--max-old-space-size=8096'
```

Comment:
Amazing

Reply:
😂

Reply:
😅  I think I might scope this to client test now to avoid leaking this detail to global config

Reply:
In next PR